autogpt_platform/backend/backend/integrations/webhooks/manager_registry_test.py

```
@@ -0,0 +1,116 @@
+    )
+
+
+def test_every_registered_manager_can_be_instantiated():
+    # Snapshot the keys first: `get_webhook_manager` calls `load_webhook_managers`
+    # again, and the SDK's patched loader re-inserts its managers into the same
+    # cached dict, which would mutate it mid-iteration.
+    failures: dict[str, str] = {}
+    for provider in list(load_webhook_managers()):
+        try:
+            get_webhook_manager(provider)
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)